app/controllers/application_controller.rb

```diff
@@ -6,4 +6,8 @@ def current_user
   def all_users
     User.all
   end
+
+  def current_post
+    Post.find(params[:id] || params[:post_id])
+  end
```

Comment:
- [ ] Please edit the `current_user` method and make it return the first user from the database as the project requirements mentioned. and Kindly delete those two method cause you don't need them here.

![image](https://user-images.githubusercontent.com/68796232/179263587-35809ef2-03d8-4001-aa30-bb815f428770.png)